app/helpers/media_objects_helper.rb

```diff
@@ -114,6 +114,31 @@ def display_rights_statement media_object
         content_tag(:dt, 'Rights Statement') + content_tag(:dd) { link }
       end
 
+      # Lending period is stored as seconds. Convert to days and hours for display.
+      def display_lending_period object
+        d, h = (object.lending_period/3600).divmod(24)
+
+        replacement = {
```

Comment:
Rails has a [pluralize helper method](https://apidock.com/rails/ActionView/Helpers/TextHelper/pluralize) that makes this easy:
```
pluralize(1, 'day') # => 1 day
pluralize(2, 'day') # => 2 days
```